Darkly/LDConfig.h

```diff
@@ -2,6 +2,8 @@
 //  Copyright © 2015 Catamorphic Co. All rights reserved.
 //
 
+@class LDUserModel;
```

Comment:
I shifted the LDUserModel.h import from .m to .h file to provide the public `withUserModel:` method. The import was creating a cyclic redundancy, so added the class reference instead of importing `LDUserModel.h`.